osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/utils/DlmsHelper.java

```diff
@@ -886,13 +886,13 @@ private void logAndThrowExceptionForUnexpectedResultData(final DataObject result
     public void validateBufferedDateTime(final DateTime bufferedDateTime, final CosemDateTimeDto cosemDateTime,
             final DateTime beginDateTime, final DateTime endDateTime) throws BufferedDateTimeValidationException {
 
-        if (bufferedDateTime == null) {
-            final DateTimeFormatter dtf = ISODateTimeFormat.dateTime();
-            throw new BufferedDateTimeValidationException(
-                    "Not using an object from capture buffer (clock=" + cosemDateTime
-                            + "), because the date does not match the given period, since it is not fully specified: ["
-                            + dtf.print(beginDateTime) + " .. " + dtf.print(endDateTime) + "].");
-        }
+//        if (bufferedDateTime == null) {
+//            final DateTimeFormatter dtf = ISODateTimeFormat.dateTime();
+//            throw new BufferedDateTimeValidationException(
+//                    "Not using an object from capture buffer (clock=" + cosemDateTime
+//                            + "), because the date does not match the given period, since it is not fully specified: ["
+//                            + dtf.print(beginDateTime) + " .. " + dtf.print(endDateTime) + "].");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine)

Reply:
done